Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/gravity.yml

```diff
@@ -0,0 +1,17 @@
+# Mini Gravity Generator
+- type: entity
+  id: GravityGeneratorMiniMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: mini gravity generator machine board
+  description: A machine printed circuit board for a mini gravity generator
+  components:
+    - type: MachineBoard
+      prototype: GravityGeneratorMini
+      requirements:
+        Capacitor: 3
+        Laser: 4
```

Comment:
Using machine parts without implementing any upgrades is a no no. Just use plasma and cables or something.

Reply:
i want to implement upgrades later on

and i changed indent since im pretty sure 3 space is "wrong", 2 space is right

Reply:
i guess can do non-component recipe until upgrading implemented